mockgen.yaml

```diff
@@ -0,0 +1,44 @@
+force: true
```

Comment:
This file is learned from https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/mockgen.yaml